### PR TITLE
6 missions page frameworkbarebones

### PR DESCRIPTION
**Pull request by Chetan559**
Mission page update

### CODE REVIEW

File: apps/frontend/public/Mission.png
Comment (knowOne08) [inline apps/frontend/public/Mission.png:1]: make the image as the background, like the video clip in  other pages

File: apps/frontend/src/app/components/Navbar.tsx
Comment (knowOne08) [inline apps/frontend/src/app/components/Navbar.tsx:1]: Ese pura delete karke change mat karo change tracking mai problem ho jata hai, it's hard to figure out what has been changed. Next time se dhyan rakhna

File: apps/frontend/src/app/pages/AboutUs.tsx
Comment (knowOne08) [inline apps/frontend/src/app/pages/AboutUs.tsx:1]: Is file mai changes kyu hai bhai ?

Agar pretttier aur esa koi formatting tool use kar rahe ho to mat karo.

File: package.json
Comment (knowOne08) [inline package.json:1]: Ye dono remove kar dena agar nahi use kar rahe hai  to.

**Review by knowOne08 — CHANGES_REQUESTED** [review]
Sahi hai.

Jo bhi changes bataye hai wo dekh ke resovle kar dena jaldi se

File: apps/frontend/public/Mission.png
Comment (Chetan559) [reply apps/frontend/public/Mission.png:1]: done

File: apps/frontend/src/app/pages/Missions.tsx
[the remark this answers is not in the document]
Comment (Chetan559) [reply apps/frontend/src/app/pages/Missions.tsx:1]: done

File: package.json
Comment (Chetan559) [reply package.json:1]: yarn.lock k conflicts baki h

**Pull request merged**